src/index.html

```diff
@@ -1,120 +1,175 @@
+          <div class="d-flex">
+            <input
+              class="form-control"
+              type="search"
+              placeholder="Filter..."
+              aria-label="Filter"
+            />
+          </div>
         </nav>
-    </header>
-    <main style="display:none">
-        <div class="container">
-            <nav aria-label="breadcrumb">
-                <ol class="breadcrumb"></ol>
-            </nav>
 
-            <!-- START CONTENT -->
-            <div id="commandbar"></div>
-            <div id="files"></div>
-            <div id="footer"></div>
-            <!-- END CONTENT -->
-        </div>    
+        <!-- START CONTENT -->
+        <div id="commandbar"></div>
+        <div id="files">
+          <div class="grid-col-4 padding-5">
+            <nav aria-label="Side navigation,">
+              <table class="table table-hover">
+                <tbody class="bucketlist"></tbody>
+              </table>
+            </nav>
+          </div>
+        </div>
+        <div id="footer"></div>
+        <!-- END CONTENT -->
+      </div>
     </main>
     <footer>
-        <div class="container">
-            <nav style="float:left;">
-                <ul>
-                    <li>
-                    <a href='http://www.nasa.gov/' target='_blank'>NASA</a>
-                    </li>
-                    |
-                    <li>
-                    <a href='http://www.caltech.edu/' target='_blank'>Caltech</a>
-                    </li>
-                    |
-                    <li>
-                    <a href='https://www.jpl.nasa.gov/copyrights.php' target='_blank'>Privacy</a>
-                    </li>
-                    |
-                    <li>
-                    <a href='https://www.jpl.nasa.gov/imagepolicy'>Image Policy</a>
-                    </li>
-                    |
-                    <li>
-                    <a href='https://www.jpl.nasa.gov/contact_JPL.php'>Feedback</a>
-                    </li>
-                </ul>
-            </nav>
-            <div class='credits'>
-                <span class='credits_manager'>Site Manager: <a href='mailto:pds-operator@jpl.nasa.gov'>PDS Help Desk</a></span>
-                <span>CL#TBD</span>
-            </div>
-        </div>    
-    </footer>    
-</body>
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6" style="text-align: left">
+            <img
+              src="./images/nasa-logo.78fcba4d.svg"
+              alt="NASA Logo"
+              style="
+                top: 0px;
+                left: 0px;
+                opacity: 1;
+                transition: filterBrightness 900ms cubic-bezier(0.4, 0, 0.2, 1)
+                    0s,
+                  filterSaturate 1200ms cubic-bezier(0.4, 0, 0.2, 1) 0s,
+                  opacity 600ms cubic-bezier(0.4, 0, 0.2, 1) 0s;
+              "
+            />
+            <p class="footerLinksTitle">PLANETARY DATA SYSTEM</p>
```

Comment:
Should this be `Planetary Data System` and the CSS class `footerLinksTitle` should have `text-transform: uppercase`?